core/unix/injector.c

```diff
@@ -154,7 +154,7 @@ static bool
 inject_ptrace(dr_inject_info_t *info, const char *library_path);
 
 static long
-our_ptrace(enum __ptrace_request request, pid_t pid, void *addr, void *data);
+our_ptrace(int request, pid_t pid, void *addr, void *data);
```

Comment:
Are the musl ptrace constants #defines and not an enum?  Should we treat like we do for Mac and create an enum so everyone has the same type?  Or at least put in a comment so we know why it's not using the proper type.

Reply:
Yeah, they're #defines. The enum solution won't be as clean as it is for Mac, since you'd have to #undef the existing constants. I'll go with a comment, I guess.